src/aviary/utils.py

```diff
@@ -284,6 +284,11 @@ class MultipleChoiceQuestion(BaseModel):
     question: str = Field(
         description="Question to answer (without multiple choice options)."
     )
+
+    question_id: str = Field(
+        default="Q", description="Unique identifier for the question."
```

Comment:
Note that the default value won't be unique, so can you update the `description` to account for this?